examples/router/src/components/pagination.rs

```diff
@@ -31,7 +31,7 @@ impl Component for Pagination {
 
     fn view(&self, ctx: &Context<Self>) -> Html {
         html! {
-            <nav class="pagination is-right" role="navigation" aria-label="pagination">
+            <nav class="pagination is-right"> // role="navigation" aria-label="pagination"
```

Comment:
Forgotten comment.

Reply:
Not a forgotten comment. See the second point mentioned under limitation

Reply:
Added a fix for this.